composer.json

```diff
@@ -83,7 +84,10 @@
         "doctrine/orm":                 "~2.3",
         "symfony/property-access":      "~2.5",
         "symfony/validator":            "~2.1",
-        "symfony/twig-bundle":          "~2.1"
+        "symfony/twig-bundle":          "~2.1",
+        "symfony/phpunit-bridge":       "2.8.*@dev",
```

Comment:
Well, this is just a bridge to handle deprecations. Not dependent to Symfony itself.

I will update it to `~2.7`.